compiler/src/iree/compiler/Codegen/Dialect/VectorExt/IR/BUILD.bazel

```diff
@@ -0,0 +1,172 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+load("//build_tools/bazel:build_defs.oss.bzl", "iree_compiler_cc_library", "iree_gentbl_cc_library", "iree_tablegen_doc", "iree_td_library")
```

Comment:
This file should be using the loaded `iree_tablegen_doc` function. That generates markdown files from tablegen files and we postprocess the generated files to host them on https://iree.dev/reference/mlir-dialects/.

I can help with that if you'd like. Some changes will need to be made across various files (searching for `IREEVectorExt` under `docs/` will show them)

Reply:
I would prefer to include the changes here.

Bazel at least should complain about the unused load, which is why I anchored on that for this comment.

The website publication workflow runs automatically after each commit is merged to `main`: https://github.com/iree-org/iree/actions/workflows/publish_website.yml. That does not run any tests on presubmit and I'm not sure how that will handle these files moving without the scripts being updated.

Here's a model PR showing where to make edits: https://github.com/iree-org/iree/pull/17743. Docs for previewing website changes locally: https://github.com/iree-org/iree/tree/main/docs/website#how-to-edit-this-documentation

Reply:
Ok, let me try to include them

Reply:
I added doc gen and checked that it produces the docs properly.